index.md

```diff
@@ -301,78 +301,54 @@ should therefore not be put in a version control repository.
+
+####Compiled manuscripts and version control
+
+This solution is technically more complicated, but used by many researchers already. 
+
+1. The manuscript is written in a plain text format such as LaTeX or Markdown that enables version control for the files involved, and converted to other formats such as PDF after major changes.
+2. Tools needed to compile manuscripts (e.g., a makefile or a LaTeX style file) are included in the project folder (and kept under version control) just like tools used to do simulation or analysis.
 
 ### Discussion
 
-We recommend against traditional desktop tools like LibreOffice and Microsoft Word
-because they make collaboration more difficult than either of our recommended alternatives:
+We recommend against traditional desktop tools like LibreOffice and Microsoft Word because they make collaboration more difficult than either of our recommended alternatives:
 
-*   If the document lives online (Google Docs),
-    then everyone's changes are in one place,
-    and hence don't need to be merged manually.
-*   If the document lives in a version control system,
-    it provides good support for finding and merging differences resulting from concurrent changes.
+*   If the document lives online (e.g. Google Docs), then everyone's changes are in one place, and hence don't need to be merged manually.
+*   If the document lives in a version control system, it provides good support for finding and merging differences resulting from concurrent changes. It also adds the possibility of using the pull-request model for suggestions, additions and pre-merge review.
 
-We also believe that researchers should use a bibliography manager of some sort,
-but discussion of those is outside the scope of this paper.
+Importantly, our recommendations clearly define the master document allowing everyone to collaborate on an equal footing. For the same reason, even if you are writing the manuscript all by yourself, we recommend using one of these strategies.
 
-[Bernhard Konrad][konrad-comment-tracking] and the lead author had this exchange:
+We also believe that researchers should use a bibliography manager of some sort, but discussion of those is outside the scope of this paper.
 
-> BK: I'm still not convinced about the necessity to track the manuscript for a single author.
->
-> GW: The reviews I'm getting on this outline, and the ease of doing pre-commit review, convinces me.
->
-> BK: That's because you have friends who care about and understand what you are writing.
-> This intersection is often empty for, say, a grad student.
->
-> GW: Maybe if we provide better tools, we can help fix that.
+As an example of an implementation of these recommendations, this paper was written using a central online repository (Github), the *issue* functionality for discussing the outline and text, and *pull requests* for reviewing the contributions from different authors and other contributors before merging them in.
```

Comment:
We should say somewhere that there should be one way to provide feedback (as well as just one way to author), i.e., settle on issues, a mail thread, or *something* as the One True Channel.